great_expectations/render/view/templates/navbar.j2

```diff
@@ -1,7 +1,7 @@
 <nav id="navigation" class="navbar navbar-light bg-light sticky">
     <nav class="nav nav-pills flex-column">
         {% if data_asset_name -%}
-            <a class="navbar-brand" href="#">{{data_asset_name}}</a>
+            <a class="navbar-brand" href="#">{{data_asset_name[2]}}</a>
```

Comment:
hard-coded index?

Reply:
Assuming data_asset_names are always triples, this isn't such a crazy assumption.

I'd like to spend a little more time thinking about the best way to set up site navigation---but not today.